utils/src/main/java/org/corfudb/utils/lock/LockClient.java

```diff
@@ -154,6 +183,30 @@ private void monitorLocks() {
         ));
     }
 
+
+    /**
+     * Stop monitoring the locks.
+     */
+    private void stopMonitorLocks() {
+        synchronized (lockMonitorFuture) {
+            if (lockMonitorFuture.isPresent() && !lockMonitorFuture.get().isDone()) {
+                boolean cancelled = lockMonitorFuture.get().cancel(false);
+                if (!cancelled) {
+                    log.error("Lock: Could not cancel the future for lease renewal!!!", lock.getLockId());
```

Comment:
cancelled can be false even if the future has already completed.  In that case, it is not an error.  Can this be changed to debug or trace?